src/Entity/CronJob.php

```diff
@@ -350,6 +350,10 @@ public function build_operations_alter(&$allowed_operations) {
    * Invoke the jobs callback.
    */
   public function invoke() {
+    if (!is_callable($this->getCallback())) {
+      \Drupal::logger('ultimate_cron')->error('The callback "@callback" for the cron job "@label" is invalid.', ['@callback' => $this->callback, '@label' => $this->getTitle()]);
+      throw new Exception('Callback invalid');
```

Comment:
We can do better with the exception message. Let's use the same message for the exception, use sprintf() for the replacements. ultimate cron has it's own per-job logging system, we should probably add it there?